wgpu-types/src/lib.rs

```diff
@@ -783,7 +783,20 @@ bitflags::bitflags! {
         /// This is a native only feature.
         const SHADER_EARLY_DEPTH_TEST = 1 << 62;
 
-        // 62..64 available
+        /// Allows two outputs from a shader to be used by blending
+        /// For more info see the OpenGL extension ARB_blend_func_extended
+        ///
+        /// Supported platforms:
+        /// - OpenGL (with ARB_blend_func_extended)
+        /// - Metal
+        /// - Vulkan (with ARB_blend_func_extended)
+        /// TBD:
+        /// - DX11 / DX 12
+        ///
+        /// This is a native only feature.
+        const BLEND_FUNC_EXTENDED = 1 << 63;
```

Comment:
```suggestion
        const DUAL_SOURCE_BLENDING = 1 << 63;
```

I think calling this dual-source blending would be preferred.